src/components/Footer/index.jsx

```diff
@@ -0,0 +1,72 @@
+import style from "./footer.module.css";
+
+const Footer = () => {
```

Comment:
This whole page has too much stuff that has nothing to do with the project remove those things